syntax = "proto3";
package cqp.remote;
import public "google/protobuf/empty.proto";

message Key {
    bytes key =  1;
    uint64 seqID = 2;
    string localID = 3;
}

message Empty {}

service KeyTransfer
{
    rpc OnKeyFromCQP(Key) returns (Empty);
    rpc OnKeyFromSatellite(Key) returns (Empty);
}
